recipe/meta.yaml

```
@@ -1,23 +1,25 @@
 {% set name = "miniopy-async" %}
-{% set version = "1.21.1" %}
+{% set version = "1.21.2" %}
+{% set python_min = "3.10" %}
```

```suggestion
{% set python_min = "3.11" %}
```

While you fixed that for `v1.21.3` with https://github.com/hlf20010508/miniopy-async/commit/c048ead2acdabb4ed9323f3dd0e7508ffa0ced64, for `v1.21.2` you still have the import, so you need to bump the `python_min` to `3.11` for this build, and then for the `v1.21.3` build you can lower it to Python 3.10 again.

Though you need to rerender the feedstock each time.

---

Got it. I'll do this for the next PR.